src/picongpu/include/simulation_defines/param/ionizerConfig.param

```diff
@@ -164,4 +165,35 @@ namespace picongpu
         28.339,
         28.339
     );
-}
+
+namespace particles
+{
+namespace ionization
+{
+
```

Comment:
I added a `particles` namespace as well - but just for the TF fitting parameters for now to not break the methods which use the atomic and effective atomic numbers.

I still had to add a lot of `thomasFermi::` though. (Unless every model gets its own namespace.)